sections/schema.js

```diff
@@ -302,8 +302,8 @@ export default [
   {
     type: "method",
     method: "datetime",
-    example: "table.datetime(name, [precision])",
-    description: "Adds a datetime column, with optional precision for MySQL.",
+    example: "table.datetime(name, [useTz], [precision])",
+    description: "Adds a datetime column, defaulting to timestamptz in PostgreSQL unless false is passed as the second argument. MySQL does not have useTz option, second argument is precision.",
```

Comment:
I find this confusing: should I explicitly check whether the DB is MySQL and omit the second argument in that case? I thought one of the points of using Knex was that it works the same regardless of DBMS (wherever possible). If this is the case, I think the code needs to be updated so that the signature is the same regardless of DBMS.

Or can I just pass `useTz` anyway and have it be ignored on everything but PostgreSQL? In that case the documentation is misleading.

Reply:
I find it confusing as well, but making API consistent here is going to introduce breaking changes. Current plan is to introduce option object support that would be consistent across different dialects, but current messy positional argument option is also going to be supported for compatibility reasons. It's going to be a documentation nightmare though 😂

Reply:
No need to keep documentation for those legacy parameters. We don't want to promote existence of those :)

Reply:
and yeah I know, it is messy and ugly...